metrics/README.md

```diff
@@ -25,3 +25,69 @@ function in that file has the ability to also `curl` or `socat` the JSON results
 by environment variables (see the file source for details). This method has been used to store results in
 Elasticsearch and InfluxDB databases for instance, but should be adaptable to use with any REST API that accepts
 JSON input.
+
+## Scaling execution
+This section describes a complete step-by-step scaling execution upto results reporting by using `scaling/k8s_scale.sh` tool which launch a series of workloads and take memory metric measurements after each launch.
+
+**Requirements**
+* A kubernetes cluster up and running (tested on v1.15.3).
+* `bc` and `jq` packages.
+* Docker (only for report generation).
+
+The steps to execute scaling framework are listed below, which need to be executed into the master node of the kubernetes cluster to avoid network issues:
+1. Clone `cloud-native-setup` repository into a preferred directory.
+2. Change directory upto `cloud-native-setup/metris`.
```

Comment:
This may sound picky, but sometimes we're very lazy, could you please put the proper commands for doing the clone, etc
```
git clone https://github.com/clearlinux/cloud-native-setup.git
cd cloud-native-setup/metrics
```

Reply:
Done :)